### ISSUE TITLE
Please add support for Nova 4

### ISSUE THREAD

**Issue by newtec-foggia** [illuminatech/nova-config#3]
Hi,

Please add support for Nova 4.
Regards

**Comment by klimov-paul** [comment]
Upgrade guide:
https://nova.laravel.com/docs/4.0/upgrade.html

**Comment by leandrogehlen** [comment]
Hello, @klimov-paul, i can provide the license key.
Can i send in your email klimov.paul@gmail.com?

**Comment by klimov-paul** [comment]
Resolved via #5

@leandrogehlen, @newtec-foggia, please check if "dev-master" version works fine with Nova 4 for you.

**Comment by leandrogehlen** [comment]
I have tested in a new application.

The error bellow  happened:

```json
{
    "message": "Method Illuminate\\Config\\Repository::restore does not exist.",
    "exception": "BadMethodCallException",
...
} 
```

i believe that right dependencies should be: 
https://github.com/illuminatech/nova-config/blob/master/composer.json#L24

` "illuminatech/config": "^1.4.0"`

and should be updated readme to works with new verison of `"illuminatech/config": "^1.4.0`

**Comment by leandrogehlen** [comment]
You are right. I forgot to register service provider.
I have an issue with disabled save button.

https://user-images.githubusercontent.com/1750751/224698502-2dc67c7a-43f4-42d6-9670-e2079b11520a.mp4

**Comment by klimov-paul** [comment]
Fixed 7fedadeec93d585facef07119b9a9c6785f5702b
